staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -42,10 +42,15 @@ import (
 type AdmissionOptions struct {
 	// RecommendedPluginOrder holds an ordered list of plugin names we recommend to use by default
 	RecommendedPluginOrder []string
-	// DefaultOffPlugins a list of plugin names that should be disabled by default
-	DefaultOffPlugins []string
-	PluginNames       []string
-	ConfigFile        string
+	// DefaultEnabledPluginSet is a set of plugin names that is enabled by default
+	DefaultEnabledPluginSet sets.String
+	// DEPRECATED flag, should use EnabledAdmissionPlugins and DisabledAdmissionPlugins.
+	// They are mutually exclusive, specify both will lead to an error.
+	PluginNames []string
```

Comment:
This is still used for the old `--admission-control` flag?

Reply:
It is, see below.

Reply:
Yes

Reply:
Not change the previous `--admission-control` behavior.

Reply:
:+1: